packages/kbn-i18n/README.md

```diff
@@ -452,20 +454,10 @@ In order to translate attributes in AngularJS we should use `i18nFilter`:
 >
 ```
 
-## Build tools
-
-In order to simplify localization process, some build tools will be added:
-- tool for verifying all translations have translatable strings
-- tool for checking unused translation strings
-- tool for extracting default messages from templates
-
-While `react-intl` has
-[babel-plugin-react-intl](https://github.com/yahoo/babel-plugin-react-intl)
-library which extracts string messages for translation, angular wrapper requires
-own implementation of such tool. In order to extracrt translation keys from the
-template, we have to parse it and create AST object. There is a
-[babel-plugin-syntax-jsx](https://github.com/babel/babel/tree/master/packages/babel-plugin-syntax-jsx)
-plugin which helps to parse JSX syntax and then create AST object. Unfortunately,
-there are no babel plugins to parse angular templates. One of the solution can be internal
-[`$parse.$$getAst`](https://github.com/angular/angular.js/blob/master/src/ng/parse.js#L1819)
-angular method.
+## I18n tools
+
+In order to simplify localization process, some build tools were implemented:
+- tool for verifying all translations have translatable strings and extracting default messages from templates
+- tool for verifying locale files and integrating them to Kibana
```

Comment:
__nit:__ `locale files` - in some places we call them, `translation files`, in others `localization files` and here `locale files`, would you mind picking just one term?